go.mod

```diff
@@ -1,23 +1,23 @@
 module github.com/crossplane/function-sdk-go
 
-go 1.24
+go 1.24.0
 
 require (
 	github.com/bufbuild/buf v1.42.0
-	github.com/crossplane/crossplane-runtime v1.18.0
+	github.com/crossplane/crossplane-runtime v1.20.0
```

Comment:
i expected this to be `v2.0.0` 😉 

Note that the module path needs updating to `github.com/crossplane/crossplane-runtime/v2` now also as per https://github.com/crossplane/crossplane/discussions/6672

Reply:
wtf did I commit there.. 😕 
I was already wondering why everything went so smooth. Will upgrade to v2 now, and see if I can test it with a function. I'm not sure how much spare time I can spend here, but I'll try my best.

Converting to draft while working on it.